Framework/Core/test/test_StringHelpers.cxx

```diff
@@ -42,4 +44,13 @@ BOOST_AUTO_TEST_CASE(StringHelpersConstStr)
   static_assert(std::is_same_v<decltype(myConstStr), ConstStr<'h', 'e', 'l', 'l', 'o', 'W', 'o', 'r', 'l', 'd'>>);
   static_assert(myConstStr.hash == (uint32_t)942280617);
   BOOST_CHECK_EQUAL(myConstStr.hash, compile_time_hash("helloWorld"));
+
+  if constexpr (is_const_str<decltype(myConstStr)>::value) {
```

Comment:
why not making it a constexpr function, rather than having it act on the type?

Reply:
I mean something like;

```c++
template <typename T>
constexpr bool is_const_str_v(T) {
  return is_const_str<T>::value;
}
```

Reply:
great idea, thanks! I will add it.